tests/console/docker_image_rpm.pm

```diff
@@ -0,0 +1,75 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test installation and running of the docker image from RPM repository for this snapshot
+# Maintainer: Fabian Vogt <fvogt@suse.com>
```

Comment:
# Maintainer: Pavel Dostál <pdostal@suse.cz>

Reply:
That is not true. This test uses docker images from RPMs which I didn't wrote - I just renamed it.

Reply:
I see, but you can be still maintainer ;)